easybuild/easyconfigs/c/CUDA-Python/CUDA-Python-12.6.2.post1-gfbf-2024a-CUDA-12.6.0.eb

```diff
@@ -20,16 +20,48 @@ dependencies = [
     ('SciPy-bundle', '2024.05'),
 ]
 
+exts_default_options = {
+    'source_urls': ['https://github.com/%(github_account)s/cuda-python/archive'],
+    'sources': [{
+        'download_filename': f'v{version}.tar.gz',
+        'filename': '%(name)s-%(version)s.tar.gz'
+    }],
+}
 exts_list = [
     ('pyclibrary', '0.2.2', {
+        'source_urls': ['https://pypi.python.org/packages/source/%(nameletter)s/%(name)s'],
+        'sources': ['%(namelower)s-%(version)s.tar.gz'],
         'checksums': ['9902fffe361bb86f57ab62aa4195ec4dd382b63c5c6892be6d9784ec0a3575f7'],
     }),
+    ('cuda-bindings', version, {
+        'modulename': 'cuda.bindings',
+        'start_dir': 'cuda_bindings',
+        'checksums': ['abbfad534cb51d79f748cc5dfdc761fa6bb2c7a2b59e64b1d03608e7b0f64b1b'],
+    }),
+    ('cuda-core', '0.3.1', {
+        'modulename': 'cuda.core',
+        'patches': ['cuda-core-%(version)s_old_license_syntax.patch'],
+        'start_dir': 'cuda_core',
+        'checksums': [
+            {'cuda-core-0.3.1.tar.gz': '14ed346e848f796929516ebce1e3b33567c4f285abf44aa767ec8e9542ca6a68'},
+            {'cuda-core-0.3.1_old_license_syntax.patch':
+             '48ae7cacc258f7cc88743f7c961b49b37d3b1fc07c1f008fd2043697d278791f'},
```

Comment:
@Micket This needs some love after merging of #24665 ?

test build now fails with
```
Checksum verification for extension patch cuda-core-0.3.1_old_license_syntax.patch failed
```